lib/facter/ethtool.rb

```diff
@@ -15,47 +15,70 @@ def ethtool(args)
   def ethtool_result_to_h(cmd_output, has_no_headlines = false)
     result = {}
     current_subsection = result
+    current_list = nil
```

Comment:
`current_list` seems to be unused. Can you remove it?

Reply:
Done